depfile.yaml

```diff
@@ -0,0 +1,153 @@
+paths:
+  - ./src
+  - ./vendor/codeigniter4/codeigniter4/system
+  - ./vendor/tatter
```

Comment:
I don't believe we're pulling in any `tatter` dependencies.

Reply:
Yep, templates got me 😂 Fix should already be in place.